h/static/scripts/directive/annotation.coffee

```diff
@@ -163,6 +163,12 @@ AnnotationController = [
         this.render()
         this.view()
 
+    this.group = ->
+      if model.group
+        group.getGroup(model.group)
+      else
+        group.focusedGroup()
```

Comment:
No upgrade path needed. Pre-groups annotations have no group field and will continue not to have one, and the code treats this as meaning that they belong to the "Public" group.

Right now you can select a group and then edit a pre-groups annotation and save it and it will get the group added, true. But once the next group request is completed then when a group is selected only annotations that already have that group will be shown. Pre-groups annotations will only show when the "Public" group is selected, and when that's selected and you save an annotation it doesn't get any groups field added.